src/ucp/api/ucp.h

```diff
@@ -807,13 +807,15 @@ typedef struct ucp_ep_params {
      * is not set, this address is mandatory for filling
      * (along with its corresponding bit in the field_mask - @ref
      * UCP_EP_PARAM_FIELD_REMOTE_ADDRESS) and must be obtained using
-     * @ref ucp_worker_get_address.
+     * @ref ucp_worker_get_address. This is not reconfigurable parameter by
```

Comment:
`This field cannot be changed by @ref ucp_ep_modify_nb` (need to fix in all places)